api_spec.yaml

```diff
@@ -322,12 +323,18 @@ paths:
                       $ref: '#/components/schemas/createJWTPayload'
                   required:
```

Comment:
in request object, in which the user gives cllient id and client secret + scopes, shouldn't they also give the mode (offline / online)? Cause otherwise how do we know if we are to issue a refresh token or not? Ore is refresh token always issued in this flow?

Reply:
Sure. we can have `mode` as an input parameter.